src/library/scala/collection/TraversableView.scala

```diff
@@ -25,6 +25,10 @@ trait TraversableView[+A, +Coll] extends TraversableViewLike[A, Coll, Traversabl
  *  `TraversableView`s work. Its definitions are generally not accessed directly by clients.
  */
 object TraversableView {
+  private[this] val reusedNoBuilder = new NoBuilder[Any]
+  private[collection] def noBuilder[A] = reusedNoBuilder.asInstanceOf[NoBuilder[A]]
+  @deprecated("use noBuilder")
```

Comment:
Seems weird to deprecate it when you can't use the replacement.

Reply:
maybe at a later stage.
removed for now

Reply:
removed for now